### ISSUE TITLE
AnnotationLine only has text at handleTwo

### ISSUE THREAD

**Issue by chriswilty** [d3fc/d3fc#1657]
I was drawing an SVG annotation line and was surprised to find that a text node is only added to handleTwo. Using decorate to add one to handleOne does not work so well, because handle texts are filled (using label property) before decorate is invoked, so then handleOne text is only filled on second render.

I see two plausible solutions:
1. Add texts to both handles, with config to switch them off/on (and existing behaviour unchanged).
2. Add example to docs for how to move the text node from handleTwo to handleOne.

Obviously option 2 is simplest and less fraught! Option 1 would mean more configuration, and sensible defaults (presumably mirror of existing defaults for handleTwo text).

**Comment by chrisprice** [comment]
Any chance you could knock together a minimal codepen example skirting the problem?

**Comment by chriswilty** [comment]
How about this:
https://codepen.io/chriswilty/pen/VwKwVjq

**Comment by chrisprice** [comment]
Am I right in thinking your example shows how to achieve the desired behaviour?

**Comment by chriswilty** [comment]
Yes, the third chart shows solution two .... courtesy of your good self, @chrisprice 😁

The second chart shows how adding a _new_ text node to handle one will not fully work, as its label content is not set on first draw. Only on resizing the chart area or the window does the label appear.
